Dockerfile

```diff
@@ -13,7 +12,7 @@ ENV KAFKA_JMX_OPTS="-Dcom.sun.management.jmxremote=true -Dcom.sun.management.jmx
 
 ADD docker/download_kafka.sh /tmp/download_kafka.sh
 
-RUN apt-get update && apt-get install -y curl python3-pip && sh /tmp/download_kafka.sh ${SCALA_VERSION} ${KAFKA_VERSION} ${KAFKA_DIR} ${JOLOKIA_VERSION}
+RUN yum -y update && yum install -y curl gzip tar python3-pip && sh /tmp/download_kafka.sh ${SCALA_VERSION} ${KAFKA_VERSION} ${KAFKA_DIR} ${JOLOKIA_VERSION}
```

Comment:
So it's actually 3 changes at once...

What would it take to still use the same base OS image (debian?), but install Corretto JDK from here: https://docs.aws.amazon.com/corretto/latest/corretto-17-ug/downloads-list.html ?